### PR TITLE
WIP vndr buildkit to e57eed4 and fsutil to b19464c

### PR DESCRIPTION
**Pull request by andrewhsu**
**- What I did**

Update vendor of buildkit to [e57eed4](https://github.com/moby/buildkit/tree/e57eed420c7573ae44875be98fa877175b4677a1) and fsutil to [b19464c](https://github.com/tonistiigi/fsutil/commit/b19464cd1b6a00773b4f2eb7acf9c30426f9df42) which are the latest code from the respective master branches.

upstream buildkit changes: https://github.com/moby/buildkit/compare/9acf51e49185b348608e0096b2903dd72907adcb...e57eed420c7573ae44875be98fa877175b4677a1

upstream fsutil changes: https://github.com/tonistiigi/fsutil/compare/8abad97ee3969cdf5e9c367f46adba2c212b3ddb...b19464cd1b6a00773b4f2eb7acf9c30426f9df42

related to https://github.com/moby/moby/pull/37582

cc @tiborvass @tonistiigi 

**- How I did it**

```
$ vi vendor.conf
$ make -f docker.Makefile shell
$ vndr github.com/moby/buildkit
$ vndr github.com/tonistiigi/fsutil
```

**- How to verify it**

```
$ make -f docker.Makefile shell
$ vndr github.com/moby/buildkit
$ vndr github.com/tonistiigi/fsutil
```

**- Description for the changelog**

* Update buildkit to e57eed4 and fsutil to b19464c

**- A picture of a cute animal (not mandatory but encouraged)**

🐞

### CODE REVIEW

File: vendor/github.com/moby/buildkit/vendor.conf [hunk 1]
```diff
@@ -4,10 +4,10 @@ github.com/pkg/errors v0.8.0
 github.com/stretchr/testify v1.1.4
 github.com/davecgh/go-spew v1.1.0
 github.com/pmezard/go-difflib v1.0.0
-golang.org/x/sys 314a259e304ff91bd6985da2a7149bbf91237993
+golang.org/x/sys 1b2967e3c290b7c545b3db0deeda16e9be4f98a2
```
Comment (thaJeztah) [inline vendor/github.com/moby/buildkit/vendor.conf]: ✅ docker/cli is already ahead; https://github.com/golang/sys/compare/ac767d655b305d4e9612f5f6e33120b9176c4ad4...1b2967e3c290b7c545b3db0deeda16e9be4f98a2

File: vendor/github.com/moby/buildkit/vendor.conf [hunk 2]
```diff
@@ -4,10 +4,10 @@ github.com/pkg/errors v0.8.0
...
 
-github.com/containerd/containerd 08f7ee9828af1783dc98cc5cc1739e915697c667
-github.com/containerd/typeurl f6943554a7e7e88b3c14aad190bf05932da84788
+github.com/containerd/containerd a88b6319614de846458750ff882723479ca7b1a1
```
Comment (thaJeztah) [inline vendor/github.com/moby/buildkit/vendor.conf]: Looks like docker/cli is behind; https://github.com/containerd/containerd/compare/08f7ee9828af1783dc98cc5cc1739e915697c667...a88b6319614de846458750ff882723479ca7b1a1

File: vendor/github.com/moby/buildkit/vendor.conf [hunk 3]
```diff
@@ -23,7 +23,7 @@ github.com/Microsoft/go-winio v0.4.7
 github.com/containerd/fifo 3d5202aec260678c48179c56f40e6f38a095738c
 github.com/opencontainers/runtime-spec v1.0.1
 github.com/containerd/go-runc f271fa2021de855d4d918dbef83c5fe19db1bdd5
-github.com/containerd/console 9290d21dc56074581f619579c43d970b4514bc08
+github.com/containerd/console 5d1b48d6114b8c9666f0c8b916f871af97b0a761
```
Comment (thaJeztah) [inline vendor/github.com/moby/buildkit/vendor.conf]: docker/cli is behind; https://github.com/containerd/console/compare/cb7008ab3d8359b78c5f464cb7cf160107ad5925...5d1b48d6114b8c9666f0c8b916f871af97b0a761

File: vendor/github.com/moby/buildkit/vendor.conf [hunk 4]
```diff
@@ -39,14 +39,14 @@ golang.org/x/time f51c12702a4d776e4c1fa9b0fabab841babae631
...
-github.com/tonistiigi/fsutil 8abad97ee3969cdf5e9c367f46adba2c212b3ddb
+github.com/tonistiigi/fsutil b19464cd1b6a00773b4f2eb7acf9c30426f9df42
 github.com/hashicorp/go-immutable-radix 826af9ccf0feeee615d546d69b11f8e98da8c8f1 git://github.com/tonistiigi/go-immutable-radix.git
 github.com/hashicorp/golang-lru a0d98a5f288019575c6d1f4bb1573fef2d1fcdc4
 github.com/mitchellh/hashstructure 2bca23e0e452137f789efbc8610126fd8b94f73b
 github.com/docker/go-connections 3ede32e2033de7505e6500d6c868c2b9ed9f169d
 github.com/docker/distribution 30578ca32960a4d368bf6db67b0a33c2a1f3dc6f
 
-github.com/tonistiigi/units 29de085e9400559bd68aea2e7bc21566e7b8281d
+github.com/tonistiigi/units 6950e57a87eaf136bbe44ef2ec8e75b9e3569de2
```
Comment (thaJeztah) [inline vendor/github.com/moby/buildkit/vendor.conf]: Looks like docker/cli is behind: https://github.com/tonistiigi/units/compare/29de085e9400559bd68aea2e7bc21566e7b8281d...6950e57a87eaf136bbe44ef2ec8e75b9e3569de2

File: vendor/github.com/moby/buildkit/vendor.conf (hunk 2 above)
Comment (andrewhsu) [reply vendor/github.com/moby/buildkit/vendor.conf]: updated with 051c4bb

File: vendor/github.com/moby/buildkit/vendor.conf (hunk 3 above)
Comment (andrewhsu) [reply vendor/github.com/moby/buildkit/vendor.conf]: updated with 0674a4d

File: vendor/github.com/moby/buildkit/vendor.conf (hunk 4 above)
Comment (andrewhsu) [reply vendor/github.com/moby/buildkit/vendor.conf]: updated with ea7f32a

**Comment by andrewhsu** [comment]
Working on a more up-to-date PR #1302

**Comment by andrewhsu** [comment]
#1302 is in which makes this PR as fashionable as parachute pants

**Pull request closed without merging**